docs/source/specifications/configuration-store.rst

```diff
@@ -1,3 +1,5 @@
+.. _ConfigurationStore:
+
 Agent Configuration Store
```

Comment:
Seems like hte other pages have a
============================================
Title
============================================

Perhaps we should keep that constistancy or change to the other way?